lib/api/v3/utilities/path_helper.rb

```diff
@@ -347,9 +347,21 @@ class << self
 
           resources :project
 
+          index :project_phase
           show :project_phase
+
+          def self.project_phases_by_project(project_id)
+            "#{project(project_id)}/project_phases"
+          end
+
+          index :project_phase_definition
```

Comment:
I think the project_phase and the project_phase_by_project route can be removed. Quick testing and searching for it in the code at least suggests this.